/ralph fix the type error:


info Building TypeScript projects to check types (For visible, though excessive, progress info you can pass --verbose)
--
info starting [tsc] > node_modules/typescript/bin/tsc -b tsconfig.refs.json --pretty
proc [tsc] x-pack/platform/test/api_integration_deployment_agnostic/apis/streams/basic.ts:122:20 - error TS18046: 'response' is of type 'unknown'.
proc [tsc]
proc [tsc] 122             expect(response.logs).to.have.property('enabled', true);
proc [tsc]                        ~~~~~~~~
proc [tsc]
proc [tsc]
proc [tsc] Found 1 error.
proc [tsc]
ERROR Error: [tsc] exited with code 2
 

<br class="Apple-interchange-newline">

--- a/x-pack/platform/test/api_integration_deployment_agnostic/apis/streams/basic.ts
+++ b/x-pack/platform/test/api_integration_deployment_agnostic/apis/streams/basic.ts
@@ -113,7 +113,11 @@ export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
           });
 
           it('Elasticsearch streams is enabled too', async () => {
-            const response = await esClient.transport.request({
+            type StreamsStatusResponse = {
+              logs: { enabled: boolean } & Record<string, unknown>;
+            } & Record<string, unknown>;
+
+            const response = await esClient.transport.request<StreamsStatusResponse>({
               method: 'GET',
               path: '/_streams/status',
             });

